### PR TITLE
User guide/multichannel

### PR DESCRIPTION
**Pull request by nnenasheva**
Multichannel docs set ready.

### CODE REVIEW

File: user_guide/accounts.rst [hunk 1]
```diff
@@ -0,0 +1,287 @@
+.. _user-guide-accounts:
+
+Accounts
+========
+
+Account records ("accounts") represent  represent a person, company or group of people you do business activities with.
```
Comment (DimaSoroka) [inline user_guide/accounts.rst]: Duplicate of "represent"

File: user_guide/accounts.rst [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+a 360-degree view of the account activity. 
+
+From this article you will learn how to create new account, understand 
+their :ref:`View page <user-guide-ui-components-view-pages>`, manage and merge existing accounts and analyse their  
+details with OroCRM reports.
+
+Accounts and Customers
+----------------------
+
+If a :ref:`Magento channel <user-guide-magento-channel>` has been created in the system, a new account is created for
```
Comment (DimaSoroka) [inline user_guide/accounts.rst]: Lets start account story with the case when no channels are configured and explain channels impact down the road

File: user_guide/accounts.rst [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+:ref:`View page <user-guide-ui-components-view-pages>` consists of several sections, namely:
+
+- **General**: general details of the account, such as its name, tags, description and all the contacts assigned for the
+  account. 
+
+.. image:: ./img/accounts/accounts_view_general.png
+
+.. note::
+
+    Once a :term:`customer identity <Customer Identity>` is assigned to an Account, all of its
```
Comment (DimaSoroka) [inline user_guide/accounts.rst]: Please test this, not sure if this is the case

File: user_guide/accounts.rst [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+.. image:: ./img/accounts/accounts_view_activities.png
+
+.. note::
+    
+    If an activity-related action was done for a customer or a contact, assigned to the account they will not be 
+    displayed. Only the activities performed directly for the account are available in the section.
+
+- **Additional Information**: details of any :term:`custom fields <Custom Field>` defined for the account.
+
+- **Sections with channel names**: each the section contain details of all the customers that are assigned to this
```
Comment (DimaSoroka) [inline user_guide/accounts.rst]: Lets explain multichannels in separate section. We should make it clear on how channels and customers are related to account.

File: user_guide/accounts.rst [hunk 5]
```diff
@@ -0,0 +1,287 @@
...
+  
+Once all the necessary fields have been defined, click the button in the right top corner of the page to save the 
+account in the system.
+
+.. hint::
+
+    You can also export and import accounts with :guilabel:`Export` and :guilabel:`Import` buttons as described in 
+    the :ref:`Export and Import Functionality <user-guide-import>` guide. 
+
+Accounts View Page
```
Comment (DimaSoroka) [inline user_guide/accounts.rst]: It is important to highlight account lifetime sales value on a view page

File: user_guide/b2b_channels.rst [hunk 6]
```diff
@@ -0,0 +1,31 @@
+
+.. _user-guide-b2b-channel:
+
+B2B Channels
+============
+
+:ref:`Channels <user-guide-channels>` of B2B type ("B2B channels") represent sources of customer-related data collected 
+in the course of business-to-business activities.
+
+The :term:`customer identity <Customer Identity>` of such channels is a B2B Customer.
```
Comment (DimaSoroka) [inline user_guide/b2b_channels.rst]: Please explain that b2b customer typically is an organization.

File: user_guide/accounts.rst (hunk 4 above)
Comment (nnenasheva) [inline user_guide/accounts.rst]: we have multichannel functionality overview doc for that

File: user_guide/accounts.rst (hunk 5 above)
Comment (nnenasheva) [inline user_guide/accounts.rst]: Added the header description

**Pull request merged**